<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="cityLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>London</string>
    </property>
   </widget>
   <widget class="QLabel" name="temperatureLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>temperatureLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidityLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>humidityLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>descriptionLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>781</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>statusLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getWeather">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixmapLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>pixmapLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
